src/azul/plugins/metadata/anvil/__init__.py

```diff
@@ -333,6 +333,12 @@ def recurse(mapping: MetadataPlugin._FieldMapping, path: FieldPath):
         result[('contents', 'files')]['file_url'] = 'files.azul_file_url'
         return result
 
+    def verbatim_pfb_entity_id(self, replica: JSON) -> str:
+        if replica['replica_type'] != 'duos_dataset_registration':
```

Comment:
```suggestion
        if replica['replica_type'] == 'duos_dataset_registration':
```

and swapping the branches would be more direct and easier to understand.